# Data path. adjust them!
common:
    image_file: "/home/allen/ooad_calib_ws/real_and_ign_data/ign_right.jpg"
    pcd_file: "/home/allen/ooad_calib_ws/real_and_ign_data/ign_mid.pcd"
    result_file: "/home/allen/ooad_calib_ws/real_and_ign_data/MultiFrame/MultiFrame8.txt"

# Camera Parameters. Adjust them!
camera:
    camera_matrix: [554.987563588087, 0.0,      192.518428910887587,
                0.0,     554.613209287033897,  122.512157520149685,
                0.0,     0.0,      1.0     ]
    # camera_matrix: [4700.987563588087141e+03, 0.0,      2.115184289108875873e+03,
    #             0.0,     4.718613209287033897e+03,  1.535121575201496853e+03,
    #             0.0,     0.0,      1.0     ]                
    dist_coeffs: [-0.00636822, 0.0028381232, 0.0020548, 0.00336049, -0.00155685]
    # dist_coeffs: [-6.3682223e-02, 2.83812328e-01, 2.054823e-03, 3.36049e-03, -1.5568534e-01]

# Calibration Parameters.!
calib:
    calib_config_file: "/home/allen/ooad_calib_ws/src/thermal_range_calib/config/ign_apartment_config.yaml"
    # calib_config_file: "/home/allen/calib_ws/src/thermal_range_calib/config/ftm_left_config.yaml"
    use_rough_calib: false # set true if your initial_extrinsic is bad
